content/en/about/concepts/diagram.md

```diff
@@ -26,3 +26,27 @@ This is a setup of Jenkins X with [Lighthouse](https://github.com/jenkins-x/ligh
 <h5>Architecture for a serverless deployment using Tekton Pipelines and Lighthouse</h5>
 </figcaption>
 </figure>
+
+
+# What gets deployed in Kubernetes?
+Below are two diagrams depicting this.  We have one diagram with Prow enabled.  The other diagram is using Lighthouse, the Prow replacement.
+
+NOTE:  In the lighthouse edition of the diagram, we also show the HashiCorp Vault deployments
+
+
+<figure>
```

Comment:
Will there be no explanatory text? I wonder how much value this diagram is w/o a bit of explanation? Also, the fonts are so small it is hard to read anything unless one zooms in multiple levels. 

I also believe we have to get away from using HTML in the Markdown files. There is a Hugo shortcode for figures.